postgres-appliance/runit/patroni/run

```diff
@@ -24,7 +24,7 @@ then
 fi
 
 # Only small subset of environment variables is allowed. We don't want accidentally disclose sensitive information
-for E in $(printenv -0 | tr '\n' ' ' | sed 's/\x00/\n/g' | grep -vE '^(KUBERNETES_(SERVICE|PORT|ROLE)[_=]|((POD_(IP|NAMESPACE))|HOSTNAME|PATH|PGHOME|LC_ALL|ENABLE_PG_MON)=)' | sed 's/=.*//g'); do
+for E in $(printenv -0 | tr '\n' ' ' | sed 's/\x00/\n/g' | grep -vE '^((AWS_EC2_METADATA_SERVICE_ENDPOINT|KUBERNETES_(SERVICE|PORT|ROLE))[_=]|((POD_(IP|NAMESPACE))|HOSTNAME|PATH|PGHOME|LC_ALL|ENABLE_PG_MON)=)' | sed 's/=.*//g'); do
```

Comment:
jFYI: We inject two envars in IPv6 clusters to make the AWS SDKs do the right thing: https://github.com/zalando-incubator/kubernetes-on-aws/blob/f2a56bb4d26eb568541e2c64edfc63f8d5f166a0/cluster/manifests/02-admission-control/config.yaml#L49-L50

Not sure if `AWS_EC2_METADATA_SERVICE_ENDPOINT_MODE` is needed here, though.

Reply:
(another tool -- wal-g I mean, with wal-e (another backup tool), it unfortunately requires manual code intervention:) )